.storybook/main.ts

```diff
@@ -2,7 +2,7 @@ import { defineMain } from "@storybook/react-vite/node";
 
 const config = defineMain({
   stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
-  addons: ["@storybook/addon-docs", "./local-preset.cjs"],
+  addons: ["@storybook/addon-docs", import.meta.resolve("./local-preset.ts")],
```

Comment:
should this have worked without wrapping in `import.meta.resolve`? It at least doesn't

Reply:
yes, we've decided this is a valid breaking change.